current/runtime/fees.md

```diff
@@ -160,6 +99,113 @@ Balances module, for example, takes a fixed fee for transferring tokens.
+    MaxOperational,
+    FixedMandatory(Weight),
+}
+```
+
+In order to delineate their purposes, the enums in this group are separated into _dispatch classes_,
+which are also defined by an enum:
+
+```rust
+pub enum DispatchClass {
+    Normal,
+    Operational,
+    Mandatory,
+}
+```
+
+### Normal Dispatches
+
+Dispatches in this class represent normal user-triggered transactions. These types of dispatches may
+only consume a portion of a block's total weight limit; this portion can be found by examining the
+[`AvailableBlockRatio`](https://substrate.dev/rustdocs/master/frame_system/trait.Trait.html#associatedtype.AvailableBlockRatio).
+Normal dispatches are sent to the [transaction pool](/current/learn-substrate/tx-pool).
+
+#### `FixedNormal`
+
+Describes a normal function that will always have the specified weight.
+
+```rust
+#[weight = SimpleDispatchInfo::FixedNormal(10)]
+pub fn some_normal_function_heavy() { some_computation(); }
+```
+
+#### `MaxNormal`
+
+This is equivalent to `SimpleDispatchInfo::FixedNormal(Weight::max_value())`.
+
+#### `InsecureFreeNormal`
+
+This means that the function has no weight; it will not contribute to block fullness at all, and no
+weight fee is applied. Although the `base_fee` and `length_fee` still need to be paid, as the name
+indicates a lack of weight fees also implies a lack of security, so developers should use care when
+when adding dispatches of this type to their runtime.
+
+```rust
+#[weight = SimpleDispatchInfo::InsecureFreeNormal]
+pub fn some_normal_function_light() { noop(); }
+```
+
+### Operational Dispatches
+
+As opposed to normal dispatches, which represent _usage_ of network capabilities, operational dispatches
+are those that _provide_ network capabilities. These types of dispatches may consume the entire weight
+limit of a block, which is to say that they are not bound by the
+[`AvailableBlockRatio`](https://substrate.dev/rustdocs/master/frame_system/trait.Trait.html#associatedtype.AvailableBlockRatio).
+Dispatches in this class are given maximum priority and are exempt from paying the `base_fee` and
+`length_fee`.
+
+#### `FixedOperational`
+
+Describes a function that will have a fixed weight and can consume the reserved operational portion
+as well.
+
+```rust
+#[weight = SimpleDispatchInfo::FixedOperational(20)]
+pub fn mission_critical_function() { some_sudo_op(); }
+```
+
+#### `MaxOperational`
+
+This is equivalent to `SimpleDispatchInfo::FixedOperational(Weight::max_value())`.
+
+### Mandatory Dispatches
+
+Mandatory dispatches will be included in a block even if they cause the block to surpass its weight
+limit. This dispatch class may only be applied to
+[inherents](/current/learn-substrate/extrinsics#Inherents) and is intended to represent functions
+that are part of the block validation process. Since these kinds of dispatches are always included
+in a block regardless of their weight, it is critical that their validation process prevents malicious
```

Comment:
Slippery pronouns. The inherent's weight or the block's weight? Ditto on validation process.

Reply:
Can you please double-check that I hit the mark here?